docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+  version: '3'
+
+  services:
+    db:
```

Comment:
Why aren't you using postgres?. That's the db we are using for the djangoplicity projects and the preferred relational database these days

Reply:
Yes, about that there is a document in the path docs/tech/developer_setup.rst which states that MySQL should be installed.

I'm not sure if django is agnostic of the db engine used

Reply:
Ah, you can also let django just use sqlite, it's the default database when no other is defined.

Reply:
In that case the db service would not even be required, right?

Reply:
Exactly, I guess we don't require it, we can just let django create a sqlite database, but Django is not entirely database agnostic, there are some features that are only available if we use Postgres, and as we would like to reproduce a similar djangoplicity environment to the one we are using in production perhaps Postgres would make more sense.

Reply:
Also, don't trust much the current docs of this repository, they seem to be pretty much outdated, better to follow how the django project's team handles its project.